WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -411,6 +411,8 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
             width = width - cellMargins;
             CGFloat height = ceilf((width / imageWidth) * imageHeight);
             return height + cellMargins;
+        } else if ([self isUploadingMedia]) {
+            return CellHeight + 2*PostFeaturedImageCellMargin;
```

Comment:
Being nitpicky, but could we add spaces around the `*` ?  And doubley nitpicky, maybe switch to `2.0` for type conformity?